extension/wallet/Cargo.toml

```diff
@@ -15,6 +15,8 @@ aes-gcm-siv = { version = "0.9", features = [ "std" ] }
 anyhow = "1"
 baru = "0.1"
 bdk = { version = "0.4", default-features = false }
+bip32 = { version = "0.2", features = [ "secp256k1" ] }
```

Comment:
Chosing the `secp256k1` feature over the `secp256k1-ffi` feature pulls in another dependency in the form of `k256` whilest `secp256k1-ffi` would result in `rust-secp256k1` which we already have in the dependency graph.

Reply:
Addressed in follow-up PR here: https://github.com/comit-network/waves/pull/193